utilities/transactions/write_prepared_txn_db.cc

```diff
@@ -110,6 +110,8 @@ static void CleanupWritePreparedTxnDBIterator(void* arg1, void* arg2) {
 
 Iterator* WritePreparedTxnDB::NewIterator(const ReadOptions& options,
                                           ColumnFamilyHandle* column_family) {
+  constexpr bool ALLOW_BLOB = false;
```

Comment:
To be consistent with the previous notation, lets follow this convention: if you use capital letters, the value is what the variable reads. e..g, ALLOW_BLOB should alway be true and we use ALLOW_BLOB or !ALLOW_BLOB depends on our use case.